cv/cv/src/cv_utils/height_map.py

```diff
@@ -0,0 +1,97 @@
+import json
+import os
+from typing import Dict, Tuple
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+HEIGHT: int = 180
+WIDTH: int = 330
+RAW_DEPTH_VALS_PATH: str = (
+    f"{os.path.dirname(os.path.abspath(__file__))}/../../config/depth_vals.json"
+)
+PROCESSED_DEPTH_VALS_PATH: str = (
+    f"{os.path.dirname(os.path.abspath(__file__))}/../../config/processed_depth_vals.json"
+)
+
+
+def load_depth_vals(depth_vals_path: str) -> Dict[str, float]:
+    with open(depth_vals_path) as f:
+        depth_vals = json.load(f)
+    return depth_vals
+
+
+def process_raw_depth_vals(raw_depth_vals: Dict[str, float]) -> np.ndarray:
+    depth_vals = np.zeros((HEIGHT, WIDTH), dtype=float)
+    points = np.zeros((HEIGHT * WIDTH, 3), dtype=float)
+    for y in range(HEIGHT):
+        for x in range(WIDTH):
+            depth = raw_depth_vals[str((x, y))]
+            depth_vals[y, x] = depth
+            points[y * WIDTH + x] = x, y, depth
```

Comment:
Is depth_vals actually used?

Reply:
No it's not. It's dead code. I should delete.